piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -77,6 +77,20 @@ def add_operator_to_cluster(self, operator_name, target_namespaces=[]):
 
         return True
 
+    def verify_operator_installed(self, operator_name, namespace):
```

Comment:
I think 'namespace' should be 'operator_namespace' for better clarity. Also the method name would be better as 'is_operator_installed' that way it would have better readability in an if statement.

Reply:
Done.